File: TouchPOS_API/Models/RPT_BalanceModels.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace TouchPOS_API.Models
{
    public class RPT_BalanceModels
    {
        public class Balance_data
        {
            public string STCODE { get; set; }    
            public string STNAME { get; set; }
            public string PCODE { get; set; }
            public string PNAME { get; set; }
            public string PGCODE { get; set; }
            public string PGNAME { get; set; }
            public Double ST_QTY { get; set; }
            public string ST_UNITCODE { get; set; }
            public string ST_UNITNAME { get; set; }
            public Double ST_COST { get; set; }
        }

        public class Fillter_Balance
        {
            public string STCODE { get; set; }
            public string PCODE1 { get; set; }
            public string PCODE2 { get; set; }
        }
    }
}

File: Blackoffice/Models/RPT_BalanceModels.vb
Imports MD
Public Class RPT_BalanceModels
    Public Class value
        Public Respon As New ResponseModels
        Public Data As List(Of Balance_data)
    End Class

    Public Class Balance_data

        Public Property STCODE As String
        Public Property STNAME As String
        Public Property PCODE As String
        Public Property PNAME As String
        Public Property PGCODE As String
        Public Property PGNAME As String

        Public Property ST_QTY As Double
        Public Property ST_UNITCODE As String
        Public Property ST_UNITNAME As String

        Public Property ST_COST As Double
    End Class

    Public Class Fillter_Balance
        Public Property STCODE As String = ""
        Public Property PCODE1 As String = ""
        Public Property PCODE2 As Object = ""

    End Class
End Class
